apps/01PigLatin.js

```diff
@@ -4,13 +4,26 @@ var assert = require('assert');
 var prompt = require('prompt');
 prompt.start();
 
-
 function pigLatin(word) {
-
-    // Your code here
-
+    var vowelIndex = 0;
+        while (!isVowel(word[vowelIndex])) {
```

Comment:
Hi Chen,

I checked my code and the indentation is not there. I think I have it
updated, but I'm uncertain.

Thank you,

Anna Paz

On Sun, May 15, 2016 at 1:14 PM, Chen Yang <notifications@github.com> wrote:

> In apps/01PigLatin.js
> <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/41#discussion_r63295742>
> :
>
> >  function pigLatin(word) {
> > -
> > -    // Your code here
> > -
> > +    var vowelIndex = 0;
> > +        while (!isVowel(word[vowelIndex])) {
>
> Anna, I want you to fix the indentation here. The rest of your code looks
> great!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/41/files/e7eeff91ff3186e644623da314958ceb0e9cbbe3#r63295742>
>